src/arc_utilities/extra_functions_to_be_put_in_the_right_place.py

```diff
@@ -1,9 +1,13 @@
-from std_msgs.msg import *
-from geometry_msgs.msg import *
-from color_mapping import *
+import math
 
+from arc_utilities.color_mapping import interpolate_hot_to_cold
+from geometry_msgs.msg import Pose, Vector3, Point
+from std_msgs.msg import ColorRGBA
 
-def make_pose((px, py, pz), (rx, ry, rz, rw)):
+
+def make_pose(position, rotation):
```

Comment:
Why do we need these functions when we can use the constructor of the ROS messages?

Reply:
Not sure, and afaik they are not used in any of the repos in our org. I'll just delete them.